### PR TITLE
testing: Add smoke tests

### PR DESCRIPTION
**Pull request by marclop**
## Motivation

Adds a smoke test under `test/smoke` for the `nodejs` agent case.

## Related issues

~Depends on #51~
Part of #54 
Part of #23

### CODE REVIEW

File: test/smoke/main.tf [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+resource "local_file" "chart_values" {
+  content  = <<EOT
+apm:
+  secret_token: ${module.ec_deployment.apm_secret_token}
+  namespaces:
+    - default
+webhookConfig:
+  agents:
+    nodejs:
```
Comment (marclop) [inline test/smoke/main.tf:42]: We can add `java` and other agents as a follow up

File: test/smoke/main.tf [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    replicas = 1
+    selector {
+      match_labels = {
+        "app" = "nodejs-demo"
+      }
+    }
+    template {
+      metadata {
+        annotations = {
+          "co.elastic.traces/agent" = "nodejs"
```
Comment (axw) [inline test/smoke/main.tf:91]: Should we use the new annotation here?

**Review by axw — APPROVED** [review]
Love it :heart_eyes:

File: test/smoke/gke.tf [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  http_load_balancing        = false
+  network_policy             = false
+  horizontal_pod_autoscaling = true
+  filestore_csi_driver       = false
+
+  node_pools = [
+    {
+      name               = "default-node-pool"
+      machine_type       = "e2-medium"
+      node_locations     = "${var.gcp_region}-b,${var.gcp_region}-c"
```
[the remark this answers is not in the document]
Comment (marclop) [reply test/smoke/gke.tf:48]: It was a C/P from the example module in the terraform registry. I'll remove these

**Pull request merged**